src/stores/useHolochainStore.js

```diff
@@ -98,8 +102,7 @@ const makeUseHolochainStore = ({ installed_app_id, app_ws_url, hc_admin_port })
         throw new Error(`Couldn't find provisioned cell with role_name ${role_name}`)
       }
 
-      if( !this.signingCredentials)
-      {
+      if(!this.isLauncher && !this.signingCredentials) {
         this.setCredentials(cellId)
       }
```

Comment:
...and if we're not supporting the pure holochain case, then we can probably just remove this case and related method.